### PR TITLE
Create ROSA cluster script

### PR DESCRIPTION
**Pull request by omrirh**
Utility script for creating rosa clusters with env var parameters

### CODE REVIEW

File: clouds/aws/utilities/rosa_cluster_driver.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        f"--region={aws_region} "
+        f"--channel-group={channel_group} "
+        f"--version={openshift_version} "
+        f"--expiration-time={cluster_expiration}Z "
+        f"--replicas={replicas} "
+        f"--compute-machine-type={compute_machine_type}"
+    )
+
+
+def main():
```
Comment (myakove) [inline clouds/aws/utilities/rosa_cluster_driver.py:32]: Let's make it a proper cli tool, please use click to get the user params not only from env.

File: clouds/aws/utilities/rosa_cluster_driver.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        openshift_version=openshift_version,
+        aws_region=aws_region,
+        channel_group=channel_group,
+    )
+
+    std = execute(
+        command=command, ocm_env=os.environ[ocm_env_url], token=os.environ[ocm_token]
+    )
+
+    LOGGER.err(f"stderr:\n{std['err']}")
```
Comment (myakove) [inline clouds/aws/utilities/rosa_cluster_driver.py:63]: Check if command passed and log the relevant err/info only

File: clouds/aws/utilities/rosa_cluster_driver.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    LOGGER.info(f"stdout:\n{std['out']}")
+
```
Comment (myakove) [inline clouds/aws/utilities/rosa_cluster_driver.py:65]: Do we want to wait for it to finished? 
I think yes, I expect to run this script and when it's done I have a working cluster.

**Review by myakove — CHANGES_REQUESTED** [review]
Move it from aws please, maybe rosa folder

File: clouds/aws/utilities/rosa_cluster_driver.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    compute_machine_type = os.getenv(key="INFRA_TYPE", default="m5.xlarge")
+    cluster_name = os.getenv(key="CLUSTER_NAME", default="msi-rosa")
+    openshift_version = os.getenv(key="OPENSHIFT_VERSION", default="4.13.4")
+    aws_region = os.getenv(key="AWS_REGION", default="us-east-1")
+    channel_group = os.getenv(key="CHANNEL", default="candidate")
+
+    command = build_command_with_params(
+        cluster_name=cluster_name,
+        cluster_expiration=cluster_expiration,
+        replicas=num_replicas,
```
Comment (rnetser) [inline clouds/aws/utilities/rosa_cluster_driver.py:52]:
```suggestion
        replicas=os.getenv(key="REPLICAS", default="2"),
```
relevant to all which are used once

File: clouds/aws/utilities/rosa_cluster_driver.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+def build_command_with_params(
+    cluster_name,
+    cluster_expiration,
+    replicas,
+    compute_machine_type,
+    openshift_version,
+    aws_region,
+    channel_group,
+):
+    return (
```
Comment (rnetser) [inline clouds/aws/utilities/rosa_cluster_driver.py:19]: `rosa create cluster` has many more options than these
please add support to all

**Review by rnetser — CHANGES_REQUESTED** [review]

**Pull request closed without merging**